build.gradle

```diff
@@ -100,7 +100,7 @@ jar {
           'Plugin-License': 'GNU LGPL 3',
           'Plugin-Version': "${project.version}",
           'Plugin-Organization': 'Michael Clarke',
-          'Sonar-Version': "${sonarqubeVersion}",
+          'Sonar-Version': "9.14.0.375",
```

Comment:
This doesn't look correct.

Reply:
This is correct. This is the Sonarqube API version, not the 'package' version that is listed on the website